Prepare for release 0.1.0 (#83)
cdevents/spec: primer.md

of events of a specific version, by looking at a single field in the context.
Examples of full event versions are:

> - `dev.cdevents.build.queued.0.1.0`

Reviewer: I think "draft" should be kept in this example 
```suggestion
- `dev.cdevents.build.queued.0.1.0-draft`
```

  identified by a change in the major version, for instance 0.1.2 -> 2.0.0

While a version of the specification is work in progress, its version is tagged

> with an extra `-draft` at the end, for instance 0.1.0.

Reviewer: "draft" should not be excluded in this example :)
```suggestion
with an extra `-draft` at the end, for instance 0.1.0-draft.
```

  - Lead time for changes: the amount of time it takes a commit to get into production
  - Deployment frequency: how often an organization successfully releases to production

## Terminology

The [CDF Interoperability Special Interest Group][sig-interop] has produced a
[table of the terms][tool-terms] used by different CI/CD tools and how they

> related to each other. The SIG is also working on distilling a recommended

Reviewer: ```suggestion
relate to each other. The SIG Interoperability is also working on distilling a recommended
```

[shared terminology][shared-terms]. CDEvents strives to adopt to the shared
terminology and collaborate with the SIG Interoperability.

> Work to align terms to those identified by the SIG will continue in upcoming

Reviewer: ```suggestion
Work to align terms to those identified by the SIG Interoperability will continue in upcoming
```
